resource.address/udp/src/test/java/org/kaazing/gateway/resource/address/udp/UdpResourceAddressFactorySpiPreferIPV4Test.java

```diff
@@ -63,4 +69,29 @@ public void startupWhenMatchingIPv6AddressesFound() throws Exception {
         factory.newResourceAddress("udp://[::1]:8000");
     }
 
+    @Test
+    public void shouldNotResolveLoopbackToIPv6IfProhibited() throws Exception {
+        // set the IPV4 flag to true
+        System.setProperty(JAVA_NET_PREFER_IPV4_STACK, "true");
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/net/doc-files/net-properties.html
The system property "java.net.preferIPv4Stack" is checked only once at JVM startup. If you are setting it in each test, it won't affect them (since they are not run in separate JVM).